resources/msi/source.wxs.erb

```diff
@@ -12,23 +12,57 @@
           Version="$(var.VersionNumber)" Manufacturer="!(loc.ManufacturerName)" UpgradeCode="$(var.UpgradeCode)">
 
     <!--
-      Define the minimum supported installer version (2.0).
-      The install should be done for the whole machine not just the current user
+      Minimum installer version (5.0) - Window 7/Server 2008 R2 and above.
+      The install scope is per machine, not the current user
     -->
-    <Package InstallerVersion="200" InstallPrivileges="elevated" Compressed="yes" InstallScope="perMachine" />
+    <Package InstallerVersion="500" InstallPrivileges="elevated"
+             Compressed="yes" InstallScope="perMachine" />
 
     <Media Id="1" Cabinet="Project.cab" EmbedCab="yes" CompressionLevel="high" />
 
-    <!-- Major upgrade -->
-    <Upgrade Id="$(var.UpgradeCode)">
-      <UpgradeVersion OnlyDetect="yes" Minimum="$(var.VersionNumber)" IncludeMinimum="no" Property="NEWERVERSIONDETECTED" />
-      <UpgradeVersion Minimum="0.0.0.0" IncludeMinimum="yes" Maximum="$(var.VersionNumber)" IncludeMaximum="yes" Property="OLDERVERSIONBEINGUPGRADED" />
-    </Upgrade>
+    <!-- Launch condition to check for minimum OS: Windows 7/Server 2008R2. -->
+    <Condition Message="!(loc.MinimumOSVersionMessage)">
+      <![CDATA[Installed OR VersionNT >= 601]]>
```

Comment:
Unless we know for a fact that Chef is completely broken on Server 2008, or even earlier, I don't want to prevent people from installing there. If it's working for those people, and they can take the benefit of future Chef releases, we want them to be able to. They're going to be much worse off if they keep using a versions of Chef with unfixed bugs.

I don't think any of our other packaging forces you to use only the versions of the platform listed in our supported platforms document.

If there's a feature we need for the installer that's only available on later Windows versions, we should weigh that trade-off directly.